src/queries/queries.js

```diff
@@ -30,3 +30,11 @@ export const buildAccountQuery = () => ({
     fetchPolicy: CozyClient.fetchPolicies.olderThan(older30s)
   }
 })
+
+export const buildGeojsonQuery = geojsonId => ({
```

Comment:
Theres is already a `buildGeoJSONQuery`, this `buildGeojsonQuery` is confusing. How about renaming `buildGeoJSONQueryById` ? And maybe rename  `buildGeoJSONQuery` into `buildGeoJSONQueryByAccountId` to remain consistent?